_posts/patterns/2016-01-28-alerts.md

```diff
@@ -0,0 +1,112 @@
+---
+
+<h3 class="hd-6 example-set-hd">Error Alert with Title and Button</h3>
+<div class="example-set">
+    <div class="alert alert-error" role="alert" aria-labelledby="alert-title-error" tabindex="-1">
+        <span class="icon alert-icon icon-exclamation-triangle" aria-hidden="true"></span>
+
+        <div class="alert-message-with-action">
+            <h3 class="alert-title" id="alert-title-error">
+                We are unable to load this chart.
+            </h3>
+        </div>
+
+        <button type="button" class="alert-action btn-neutral btn">Read More <span class="sr-only">about charts</span></button>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Success Alert with Title and Message</h3>
+<div class="example-set">
+    <div class="alert alert-success" role="alert" aria-labelledby="alert-title-success" tabindex="-1">
+        <span class="icon alert-icon icon-check" aria-hidden="true"></span>
+
+        <div class="alert-message">
+            <h3 class="alert-title" id="alert-title-success">
+                We've recieved your essay submission!
+            </h3>
+
+            <div class="alert-copy">
+                Hang tight while the rest of the class catches up.  We'll let you know when your peers have
+                graded your essay and when you can help grade theirs as well.  In the meantime,
+                <a href="http://example.com">check out what's next in the course</a>.
+            </div>
+        </div>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Warning Alert with Title, Message, and Button</h3>
+<div class="example-set">
+    <div class="alert alert-warning" role="alert" aria-labelledby="alert-title-warning" tabindex="-1">
+        <span class="icon alert-icon icon-exclamation-triangle" aria-hidden="true"></span>
+
+        <div class="alert-message-with-action">
+            <h3 class="alert-title" id="alert-title-warning">
+                Waiting for Peer Responses
+            </h3>
+
+            <p class="alert-copy">
+                All submitted peer responses have been assessed.  Check back later to see if
+                more students have submitted responses.  You'll receive your grade after you've
+                completed all the steps for this problem and your peers have assessed your
+                responses.
+            </p>
+        </div>
+
+        <button type="button" class="alert-action btn-neutral btn">Reload <span class="sr-only">about assessments</span></button>
+    </div>
+</div>
+
+
+<h3 class="hd-6 example-set-hd">Warning Alert with Message Only and No Icon</h3>
```

Comment:
I'm not sure about this use case. Is the intention that this is an alert that the user shouldn't pay much attention to? It seems too subtle to me, in that it is only the very thin yellow top border that distinguishes this from a non-alert. I would be tempted to add the warning icon back, but leave it without a title so that the text doesn't look like a heading. In fact, I then wonder whether the first example should use a message and not a title.